sqlpage/templates/shell.handlebars

```diff
@@ -1,178 +1,267 @@
+                            </a>
+                        </li>
+                        {{/if}}
+                        {{/each}}
+                    </ul>
+                </div>
+            </div>
+        </aside>
+        {{else}}
         <nav class="navbar navbar-expand-md navbar-light{{#if fixed_top_menu}} fixed-top{{/if}}">
-        <div class="container-fluid gap-2">
-            <a class="navbar-brand" href="{{#if link}}{{link}}{{else}}/{{/if}}">
-                {{#if image}}
-                    <img src="{{image}}" alt="{{title}}" width="32" height="32"
-                        class="navbar-brand-image">
-                {{/if}}
-                {{#if icon}}
+            <div class="container-fluid gap-2">
+                <a class="navbar-brand" href="{{#if link}}{{link}}{{else}}/{{/if}}">
+                    {{#if image}}
+                    <img src="{{image}}" alt="{{title}}" width="32" height="32" class="navbar-brand-image">
+                    {{/if}}
+                    {{#if icon}}
                     {{~icon_img icon~}}
+                    {{/if}}
+                </a>
+                {{#if title}}
+                <h1 class="mb-0 fs-2 text-truncate">
+                    <a class="navbar-brand" href="{{#if link}}{{link}}{{else}}/{{/if}}">{{title}}</a>
+                </h1>
                 {{/if}}
-            </a>
-            {{#if title}}<h1 class="mb-0 fs-2 text-truncate"><a class="navbar-brand" href="{{#if link}}{{link}}{{else}}/{{/if}}">{{title}}</a></h1>{{/if}}
-            <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
-                data-bs-target="#navbar-menu" aria-controls="navbar-menu" aria-expanded="false"
-                aria-label="Toggle navigation">
-                <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbar-menu">
-                <ul class="navbar-nav ms-auto">
-                    {{~#each (to_array menu_item)~}}
-                        {{~#if (or (eq (typeof this) 'object') (and (eq (typeof this) 'string') (starts_with this '{')))}}
-                            {{~#with (parse_json this)}}
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbar-menu"
+                    aria-controls="navbar-menu" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse" id="navbar-menu">
+                    <ul class="navbar-nav ms-auto">
+                        {{#each (to_array menu_item)}}
+                        {{#if (or (eq (typeof this) 'object') (and (eq (typeof this) 'string') (starts_with this
+                        '{')))}}
+                        {{#with (parse_json this)}}
+                        {{#if (or (or this.title this.icon) this.image)}}
```

Comment:
I'm a little afraid of duplicating this logic... Do you think we could use the same markup for the horizontal and the vertical sidebars, with only classes differing between the two ? Or would that be impossible ?